config/jobs/kubernetes-sigs/sig-windows/release-1.27-windows.yaml

```diff
@@ -0,0 +1,57 @@
+presets:
+- labels:
+    preset-capz-windows-common-127: "true"
+  env:
+  - name: "KUBERNETES_VERSION"
+    value: "latest-1.27"
+  - name: E2E_ARGS
+    value: "-kubetest.use-ci-artifacts"
+  - name: WINDOWS
+    value: "true"
+  - name: TEST_WINDOWS #temp to unblock failing jobs on windows while we fix this in capz (https://github.com/kubernetes/kubernetes/issues/116474)
+    value: "true"
+  - name: AZURE_NODE_MACHINE_TYPE
+    value: "Standard_D4s_v3"
+periodics:
+- name: ci-kubernetes-e2e-capz-windows-1-27
+  decorate: true
+  decoration_config:
+    timeout: 4h0m0s
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-azure
+    base_ref: release-1.7
+    path_alias: sigs.k8s.io/cluster-api-provider-azure
+    workdir: true
+  - org: kubernetes-sigs
+    repo: cloud-provider-azure
+    base_ref: release-1.26
```

Comment:
does this needs to match k/k release?

Reply:
Let me drop this.
We only need `cluod-provider-azure` if we are using CAPZ release-1.8.
I was going to update all the jobs other release jobs after this merges and we can fix this one up at the same time.

Reply:
fixed